### DISCUSSION TITLE
I keep getting rejected for GitHub students

### DISCUSSION THREAD

**Discussion by alplebani** [community/community#142153] [GitHub Education]
### Hi everyone,

I provided a university ID picture and it got rejected, asking me to prove why I was in a different country. I then provided a document signed by my university, stating that I am a student at the University and explaining why I am not currently on campus. Nevertheless, I got rejected again, and I was asked again to upload the documents using the webcam (I cannot do it since the documents were sent to me by email) and then with a picture explaining why I wasn't on campus. Anyone has any idea on how to solve this?

Thanks,
Alberto

**Comment by Aleko1243** [comment]
same here.. why isn't there a way to just upload files instead of needing to take a silly picture?

**Reply by candytripn** [reply]
has rejected my school's id, official transcripts, schedules and more. Worse, it tries to tell me that my school require an email my school doesn't use! (school's email is @my.yosemite.edu but its asking for mjc,edu which doesn't exist! Getting frustrated having every official document rejected.

**Reply by candytripn** [reply]
6th try got it, i stacked everything, specifically making sure there was a matched name (case specific), school name, and recent date, and it finally accepted it.
